test/test_utils/test_query_utils.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from src.pandasaurus.utils.query_utils import chunks, get_prefixes, run_sparql_query
+
+
+def test_run_sparql_query():
```

Comment:
Can we ensure code is throwing separate exceptions for Ubergraph connection problem, invalid sparql and timeout cases with proper messages? Can we add test cases for these exceptional scenarios?